gtsfm/runner/gtsfm_runner_base.py

```diff
@@ -87,29 +81,9 @@ def construct_argparser(self) -> argparse.ArgumentParser:
         parser.add_argument(
             "--max_frame_lookahead",
             type=int,
-            default=20,
+            default=None,
```

Comment:
I kept it primarily just so I didn't have to change the structure of the CI. If you look at the code, it can still be used if the sequential retriever is specified to overwrite the value in the config. Otherwise, it will not be used.